docs/source/tutorials/first_kagome_chain.md

```diff
@@ -4,20 +4,24 @@ In this tutorial you will learn how to execute Kagome-based Polkadot-host chain
 
 ### Prerequisites
 
-1. Kagome validating node binary built as described [here](https://kagome.readthedocs.io/en/latest/overview/getting_started.html#build-full-validating-node).
-2. For your convenience make sure you have this binary included into your path:
-```
-PATH=$PATH:build/node/kagome_validating/
-```
-3. Python 3 installed in the system  
+1. [Docker](https://docs.docker.com/get-docker/) installed in the system
+2. Python 3 installed in the system  
 
 ### Tutorial
 
 During this tutorial we will:
-1. Launch Kagome network using prepared genesis file
-2. Query the balance of Alice account
-3. Send extrinsic that transfers some amount of currency from Alice to Bob
-4. Query again the balance of Alice to make sure balance was updated
+1. Get Kagome docker image
+2. Launch Kagome network using prepared genesis file
+3. Query the balance of Alice account
+4. Send extrinsic that transfers some amount of currency from Alice to Bob
+5. Query again the balance of Alice to make sure balance was updated
+
+#### Get Kagome docker image
+
+In order to run Kagome peer as a single instance in Docker, you should pull the image for Kagome first:
```

Comment:
I think that the meaning of a peer is not quite clear here, because a peer is someone on the opposite side of  a connection, but we don't have, like, our host yet and thus no 'connection' or 'network' which may have a peer in this context.

Reply:
IMO peer is the node in this context. The whole network can contain the only node. So there will be no connections at all.

Reply:
Yeah, and I mean in this case there's no peers, just a node.